app/code/Magento/Email/Api/BackendTemplateRepositoryInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Email\Api;
+
+interface BackendTemplateRepositoryInterface
+{
+    
+    /**
+     * @param \Magento\Framework\Mail\TemplateInterface $template
+     *
+     * @return mixed
+     */
+    public function save(\Magento\Framework\Mail\TemplateInterface $template);
```

Comment:
Only scalars, Data interfaces or arrays of these two are allowed as parameter and return types. Please introduce data interface for the model and use it here.